zio/parquetio/reader.go

```diff
@@ -14,11 +14,14 @@ import (
 	"github.com/xitongsys/parquet-go/source"
 )
 
+const BUFSIZE = 1000
```

Comment:
```suggestion
const bufsize = 1000
```
No need to export this, but if you do, call it Bufsize or BufSize.